change/office-ui-fabric-react-87af3408-8c21-4731-8137-0f19f77075e1.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "minor",
+  "packageName": "office-ui-fabric-react",
```

Comment:
You should add `"comment": "allow custom filter icons for the details header"` to this change file and delete the other one.

Reply:
Done!